liveRecorder/serviceWrappers/linux/liveRecorder

```diff
@@ -24,13 +24,24 @@
 
 source /etc/rc.d/init.d/functions
 . /opt/rh/python27/enable
+
 APP_NAME=liveRecorder
 SCRIPT_DIR=$(dirname $(readlink -f $0))
 APPDIR=${SCRIPT_DIR}/../../
 NAME=${APP_NAME}
 APPLICATION_PATH=${APPDIR}/main.py
-LOG_DIR="/var/log/${APP_NAME}"
-PID_DIR="/var/run"
+LOG_DIR=@LOG_DIR@
+PID_DIR=@RUN_DIR@
+
+#default values
+if [ "${LOG_DIR}" = "@LOG_DIR@" ]; then
```

Comment:
This doesn't make any sense. Either your transform this file into a template file from which the init will be generated or you use ENV vars as I explained in my comment here:
https://github.com/kaltura/liveDVR/pull/502/files/41168d211ee18133c668f72f7f9a00efb892d51e#r128896835